### PR TITLE
Updating the Logic for removing Installation Folder

### PR DESCRIPTION
**Pull request by puneet0191**
@javigomez I tried using this
```
$I->seeElement(['xpath' => "//input[@value='Installation folder successfully removed']"]);
```
But it seems Codeception is not identifying the new input element.

### CODE REVIEW

**Comment by puneet0191** [comment]
This would help us make sure if there was any issue while removing the installation folder the test would fail.

File: src/JoomlaBrowser.php [hunk 1]
```diff
@@ -143,10 +143,10 @@ public function installJoomla()
         // Wait while Joomla gets installed
         $this->debug('I wait for Joomla being installed');
         $I->waitForText('Congratulations! Joomla! is now installed.', 60, ['xpath' => '//h3']);
-        $this->debug('Joomla is now installed');
-        $I->see('Congratulations! Joomla! is now installed.',['xpath' => '//h3']);
 		$this->debug('Removing Installation Folder');
 		$I->click(['xpath' => "//input[@value='Remove installation folder']"]);
+		$this->debug('Joomla is now installed');
```
Comment (javigomez) [inline src/JoomlaBrowser.php]: removing the folder may take a few seconds I would add here a waitforElement

Comment (puneet0191) [inline src/JoomlaBrowser.php]: This is what we get when we add a waitForElement
image

Comment (javigomez) [inline src/JoomlaBrowser.php]: merging for the moment. I'll check about this locator.

**Pull request merged**